README.md

```diff
@@ -10,7 +10,7 @@ The CH32V305 and CH32V307 are interconnected microcontrollers, based on 32-bit R
 <img src="image/frame.jpg" alt="frame" style="zoom:50%;" />
  
 ### Features
-- RISC-V4F processor, max 144MHz system clock frequency;
+- RISC-V4F processor, max 144MHz system clock frequency, RV32IMAFC instruction set;
```

Comment:
This is named riscv32imafc-unknown-none-elf in Rust.
RISC-V4F already denotes an "F"-extension.